### PR TITLE
Add module paddle.v2.dataset.penn_treebank.py

### PR DESCRIPTION
**Pull request by wangkuiyi**
An initial draft of penn_treebank.py for online book chapter http://book.paddlepaddle.org/word2vec/

### CODE REVIEW

File: python/paddle/v2/dataset/penn_treebank.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+def reader_creator(filename, download_progress_handler):
+    if not tarball_exists():
+        fetch_tarball(download_progress_handler)
+
+    def reader():
+        tarf = tarfile.open(CACHE_FILENAME)
+        f = tarf.extractfile(filename)
+        for l in f:
+            print l.split()
```
Comment (helinwang) [inline python/paddle/v2/dataset/penn_treebank.py]: maybe `yield` instead of `print` here?

**Comment by reyoung** [comment]
When I implement Conll05 dataset, I found an excellent Python package named [NLTK](http://www.nltk.org/), which can get most popular datasets in language processing.

NLTK handle the dataset downloading, parsing jobs. This package is well tested by NLP experts and production ready.

The documentation is [here](http://www.nltk.org/howto/corpus.html).

For example, to use/download treebank dataset, the example code are

```python
from nltk.corpus import treebank
for field in treebank.fileids():
    for word in treebank.words(field):
       pass
```

File: python/paddle/v2/dataset/penn_treebank.py (hunk 1 above)
Comment (wangkuiyi) [inline python/paddle/v2/dataset/penn_treebank.py]: Thanks!!

**Comment by reyoung** [comment]
Closed because we decide use NLTK to handle nlp dataset.

**Pull request closed without merging**